Meson throws errors about EOT char

https://github.com/elementary/switchboard-plug-printers/blob/97a1fb6d177cc7bf1519e4fbd35b0416f3b2e0f8/src/Objects/Printer.vala#L266

When running the `pot` target:
```
src/Objects/Printer.vala:266: warning: missing context for keyword 'dpgettext'
```

When running the `po` target:
```
[0/1] Running external command printers-plug-update-po.
src/Objects/Printer.vala:266: warning: missing context for keyword 'dpgettext'
/home/daniel/Projects/switchboard-plug-printers/po/printers-plug.pot:560: context separator <EOT> within string
msgmerge: found 1 fatal error
Traceback (most recent call last):
  File "/usr/bin/meson", line 37, in <module>
    sys.exit(main())
  File "/usr/bin/meson", line 34, in main
    return mesonmain.run(sys.argv[1:], launcher)
  File "/usr/share/meson/mesonbuild/mesonmain.py", line 322, in run
    sys.exit(run_script_command(args[1:]))
  File "/usr/share/meson/mesonbuild/mesonmain.py", line 278, in run_script_command
    return cmdfunc(cmdargs)
  File "/usr/share/meson/mesonbuild/scripts/gettext.py", line 111, in run
    return update_po(src_sub, options.pkgname, langs)
  File "/usr/share/meson/mesonbuild/scripts/gettext.py", line 74, in update_po
    subprocess.check_call(['msgmerge', '-q', '-o', pofile, pofile, potfile])
  File "/usr/lib/python3.6/subprocess.py", line 291, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['msgmerge', '-q', '-o', '/home/daniel/Projects/switchboard-plug-printers/po/aa.po', '/home/daniel/Projects/switchboard-plug-printers/po/aa.po', '/home/daniel/Projects/switchboard-plug-printers/po/printers-plug.pot']' returned non-zero exit status 1.
FAILED: meson-printers-plug-update-po 
/usr/bin/python3 /usr/bin/meson --internal commandrunner /home/daniel/Projects/switchboard-plug-printers /home/daniel/Projects/switchboard-plug-printers/build po /usr/bin/python3 /usr/bin/meson /usr/bin/python3 /usr/bin/meson --internal gettext update_po --pkgname=printers-plug --extra-args=--directory=/home/daniel/Projects/switchboard-plug-printers@@--from-code=UTF-8
ninja: build stopped: subcommand failed.

```

--- a/src/Objects/Printer.vala
+++ b/src/Objects/Printer.vala
@@ -263,7 +263,7 @@ public class Printers.Printer : GLib.Object {
             unowned string reason = state_reasons;
             for (int i = 0; i < reasons.length; i++) {
                 if (reasons[i] in reason) {
-                    return dpgettext(Build.GETTEXT_PACKAGE, "printer state" + "\004" + statuses[i], "printer state".length + 1);
+                    return dpgettext2 (Build.GETTEXT_PACKAGE, statuses[i], "printer state");
                 }
             }
 

